client/state/jetpack-connect/actions.js

```diff
@@ -76,17 +76,19 @@ export default {
 		};
 	},
 
-	checkUrl( url, isUrlOnSites, flowType ) {
+	checkUrl( url, isUrlOnSites, selectedPlan ) {
 		return ( dispatch ) => {
 			if ( _fetching[ url ] ) {
 				return;
 			}
+			const action = {
```

Comment:
Yes, that's exactly what I meant. With one small difference: you'll need to export this function in order to add tests for it.